pkg/cri/server/container_create.go

```diff
@@ -183,7 +182,9 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 
 	log.G(ctx).Debugf("Container %q spec: %#+v", id, spew.NewFormatter(spec))
 
-	snapshotterOpt := snapshots.WithLabels(snapshots.FilterInheritedLabels(config.Annotations))
```

Comment:
Do we not pass annotations as snapshot labels anymore?

Reply:
I see the reasoning is that the only label names that passed this check either have two `/`, or are `containerd.io/snapshot.ref`. The first seems valid as long as we decide to not care about CRI API outside of Kubelet (a decent decision IMO). The second we should double check to see if anyone is doing.

Reply:
Right, I got rid of it because for actual Kubernetes usage it was doing nothing, and to be more specific we'd never even get as far as Containerd as an annotation in the containerd.io/snapshot/* form would fail validation. It seemed sane to me to not leave in something that would only take effect for crictl/some other cri client that isn't the Kubelet itself.

Reply:
And thats a good point.. let me check

Reply:
Ahh, the label is set by the client itself during unpack. I think we're ok here. Don't see anybody passing this as an annotation to be set for the rootfs snapshot (which wouldn't make sense afaik)  https://github.com/containerd/containerd/blob/42c6be80e34e06e62d834fac489822c0deeee6c1/pkg/unpack/unpacker.go#L297

Reply:
I think this is safe enough since k8s doesn't seem to provide a mechanism for the user to set annotations directly on the container spec, and even if they it seems unlikely someone would be manually setting `containerd.io/snapshot.ref` with a correct chain ID and depending on this behavior. So while technically a breaking change this seems okay to me. Let's leave this thread open to see if another maintainer agrees though.

Reply:
sgtm